bash_completion

```diff
@@ -2523,68 +2548,71 @@ __load_completion()
         dirs+=(./completions)
     fi
 
-    # 3) From bin directories extracted from $(realpath "$cmd") and PATH
-    local ret
-    _comp_realcommand "$1" && paths=("${ret%/*}") || paths=()
+    # 3) From bin directories extracted from path to command, and $PATH
+    [[ $cmd == */* ]] && paths=("${cmd%/*}") || paths=()
```

Comment:
I believe we still need to use the symlink-resolved `readlcmd` here. We wanted to find `share/bash-completion/completions` of the actual command even if it is called from a symlink. This is independent of how we resolve `cmd`.

Reply:
> Unsure if this check should be `== /*` instead of `== */*`. But I suppose it doesn't make much of a difference.

A relative path containing a slash can happen here only when the executable file does not exist. The question is whether we should search the corresponding completion file in the relative-path directory. I'm not sure it's worth it but added 301da7f6.

> I believe we still need to use the symlink-resolved `readlcmd` here. We wanted to find `share/bash-completion/completions` of the actual command even if it is called from a symlink. This is independent of how we resolve `cmd`. (by @akinomyoga)

I have pushed a fix for the real path bbe1ae88.

Reply:
Hm, I'm unable to locate the mentioned commits in the repo. Were they pushed to a fork somewhere? Feel free to push them in the branch here.

Reply:
Ah, sorry. It's just my mistake. I intended to push them into the PR branch but have actually pushed them into the branch of the same name in my repository. I now pushed them.